README.md

```diff
@@ -26,6 +26,12 @@ This repository contains the front-end code for the Seventh Day Adventist projec
 
 This creates all patterns, the styleguide, and the pattern lab site as well as a local server for development.
 
```

Comment:
Can we add a line about the include style? Would this be the appropriate place to do that?